### ISSUE TITLE
Net nodes cannot be moved passed other nodes

### ISSUE THREAD

**Issue by richardabendroth** [flandreas/antares#595]
When moving a wire that is connected to a net, its connection node cannot be moved passed other nodes. 

![image](https://github.com/flandreas/antares/assets/25486087/c5b651e3-f7bd-437b-865a-00170a5d7e55)
![image](https://github.com/flandreas/antares/assets/25486087/b2fc4302-01df-4740-a123-c4b5ab798ca0)

**Comment by flandreas** [comment]
Yes, that's a limitation of how the wires are designed technically. I can try to see if this can be overcome, but I guess it won't be easy.
